bin/templates/cordova/lib/build.js

```diff
@@ -409,7 +442,29 @@ function parseOpts(options, resolvedTarget) {
             console.warn('Build option \'' + options[i] + '\' not recognized (ignoring).');
         }
     }
+    if (packageArgs.keystore && packageArgs.alias) {
+        var keystore = path.relative(ROOT, path.resolve(packageArgs.keystore));
+        ret.packageInfo = new PackageInfo(keystore, packageArgs.alias, packageArgs.keystorePassword,
+            packageArgs.password, packageArgs.keystoreType);
+    } else if (fs.existsSync(path.join(CORDOVAROOT, BUILD_CONFIG_FILE))) {
```

Comment:
We've made it this far without platforms having CLI-specific code, so I think we should not check at this file location. Instead, we could have "cordova prepare" create -signing.properties files, or have "cordova run" pass along all the signing args as CLI arguments. Could even pass it along as a single argument that points to the JSON file, but I don't think we should look for it here directly.